system/src/Grav/Common/Page/Medium/MediumFactory.php

```diff
@@ -119,8 +119,8 @@ public static function scaledFromMedium($medium, $from, $to)
         }
 
         $ratio = $to / $from;
-        $width = (int) ($medium->get('width') * $ratio);
-        $height = (int) ($medium->get('height') * $ratio);
+        $width = $medium->get('width') * $ratio;
```

Comment:
This isn't directly related to the PR, but I've previously found that downscaled images sometimes got a white stripe at the far right edge. Not rounding the pixel values sent to Gregwar/Image fixed that